google/node_config.go

```diff
@@ -111,9 +111,9 @@ func schemaNodeConfig() *schema.Schema {
 					Type:        schema.TypeMap,
 					Optional:    true,
 					Computed:    true,
-					ForceNew:    true,
+					ForceNew:    false,
```

Comment:
We don't need to explicitly set this to `false` here, removing `ForceNew` should suffice.